staff/clara-rufi/e-commerce-template/ecommerce.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+body{
+    margin: 0px;
```

Comment:
if you set it to 0 you don't need to ad px. you could write margin: 0;

Reply:
ok, thanks